content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -317,6 +317,43 @@ product form.
 
 .. _sales/advanced-price-rules:
```

Comment:
This custom anchor needs to be moved, since this new **Discounts** section now comes before the **Advanced price rules** section. Please move the anchor to it's section (**Advanced price rules**).

May as well add an anchor for Discounts, in case you (or others) want to reference this brand new section!
```suggestion
.. _sales/discounts:
```